src/main/java/com/backbase/ct/bbfuel/client/legalentity/LegalEntityPresentationRestClient.java

```diff
@@ -31,20 +32,16 @@ public void init() {
         setInitialPath(config.getDbsServiceNames().getLegalentity() + "/" + CLIENT_API);
     }
 
-    public Response retrieveLegalEntityByExternalId(String externalLegalEntityId) {
+    public LegalEntityByExternalIdGetResponseBody retrieveLegalEntityByExternalId(String externalLegalEntityId) {
         return requestSpec()
-            .get(String.format(getPath(ENDPOINT_EXTERNAL), externalLegalEntityId));
-    }
-
-    public LegalEntityByIdGetResponseBody retrieveLegalEntityByLegalEntityId(String internalLegalEntityId) {
```

Comment:
Looks like this method was removed. What's the reason for that?

Reply:
method wasnt used anywhere and used the same name I would need to follow the naming scheme, assumed it was a placeholder.
I can always put the method back and change the name of my added method?

Reply:
No reason to remove an existing public method